docs/production-deployment/cloud/nexus/use-cases.mdx

```diff
@@ -0,0 +1,83 @@
+
+1. Creating a[Nexus Endpoint](/nexus/registry-endpoints#nexus-endpoints) in the [Nexus Registry](/nexus/registry-endpoints#nexus-registry) that:
+   1. Targets the handler Namespace.
+   2. Allows the caller Namespace.
+2. Creating a [Nexus Service](/nexus/services-operations) in a Worker within a handler Namespace
+   1.  Abstract Workflows with Nexus Operations [using Temporal SDK builder functions for Nexus Operations](/cloud/nexus#build-and-use-nexus-services).
+   2.  Register the Nexus Service with the Worker
+   2.  Ensure the Worker is polling the Endpoint's Task Queue. 
+3. Calling the Nexus Service from a Workflow in a different Namespace
+   1. Execute a Nexus Operation from a caller Workflow [using the Temporal SDK](/cloud/nexus#build-and-use-nexus-services).
+
+## Decompose a Monolithic Namespace
+
+When teams operate in the same monolithic Namespace, everything is available to everyone, and mis-behaving workers can trigger rate limits that affect all teams operating in that monolithic Namespace.
+Nexus enables each team to have their own Namespace for improved security, troubleshooting, and fault isolation.
+
+Temporal Nexus enables a modular application design that can evolve as you grow.
+Using Nexus Endpoints in a monolithic Namespace decouples callers from handlers, so Nexus Services can be moved to different Namespaces with minimal config changes and no downtime.
+
+**Follow these steps to decompose a large monolithic Namespace:**
+
+1. Hide service implementations behind a Nexus Endpoint.
+   1. Add [Nexus Endpoints](/nexus/registry-endpoints#nexus-endpoints) to the [Nexus Registry](/nexus/registry-endpoints#nexus-registry), with monolithic Namespace as the target.
+   2. Add [Nexus Services](/nexus/services-operations) to the same Workers as the Temporal primitives being abstracted.
+2. Service consumers use the Nexus Endpoint instead of the underlying implementation.
+   1. This can be done incrementally, until there are no direct caller dependencies on the underlying service implementations, that is the underlying Temporal Workflow and Activity primitives.
+3. Move service implementations to a different Namespace.
+   1. Create a new Namespace.
+   2. Add a Worker deployment with the Nexus Service.
+   3. Update the Nexus Endpoint target to the new Namespace.
+   4. Configure the Endpoint allowlist to allow calls from the original monolithic Namespace.
+   5. New Nexus Operations will be routed from callers in the monolithic Namespace to the new Namespace.
+4. Quiesce [Nexus Operations](/nexus/services-operations#nexus-operations) on the previous target Namespace.
+   1. Leave the previous Worker deployment running until all existing Nexus Operations in the previous Namespace have completed.
+      This includes their underlying Workflows, if any.
+   2. Previous Workers in the monolithic Namespace can have the service implementation removed, since it is now being served from the new Endpoint target Namespace.
```

Comment:
This is too risky, we should just tell users to create a new endpoint and direct calls to it while maintaining the old one until they're certain it will not get any more calls.
E.g. cancellation or other operations that might want to signal existing workflows.

Reply:
Removed for now.